### PR TITLE
feat: drop unnecessary db check on ingest

### PR DESCRIPTION
**Pull request by jobulcke**

### CODE REVIEW

File: ldes-server-infra-postgres/postgres-ingest-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/postgres/MemberPostgresRepository.java [hunk 1]
```diff
@@ -50,24 +57,14 @@ public List<IngestedMember> insertAll(List<IngestedMember> members) {
...
 					})
 					.toList();
-
-			jdbcTemplate.batchUpdate(sql, batchArgs);
-
-			return members;
-		} else {
-			return List.of();
+			return Arrays.stream(jdbcTemplate.batchUpdate(sql, batchArgs)).sum();
+		} catch (DuplicateKeyException e) {
```
Comment (Yalz) [inline ldes-server-infra-postgres/postgres-ingest-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/postgres/MemberPostgresRepository.java:61]: I don't see an explicit test that tests this edge case?

**Review by Yalz — APPROVED** [review]
